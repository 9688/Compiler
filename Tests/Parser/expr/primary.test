_asdda;
(a + b);
"sdsads";
123213;
's';
