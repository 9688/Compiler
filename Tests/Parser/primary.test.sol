"ROOT"
    "EXPR"
         "_asdda"
    "EXPR"
         "EXPR"
              "+"
                   "a"
                   "b"
    "EXPR"
         "sdsads"
    "EXPR"
         "123213"
    "EXPR"
         "s"
