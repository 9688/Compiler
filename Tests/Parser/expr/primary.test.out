
list
     _asdda
list
     list
          "+"
               a
               b
list
     sdsads
list
     123213
list
     s

